src/libs/base/base.pro

```diff
@@ -4,7 +4,7 @@
 
 TEMPLATE = lib
 TARGET = base
```

Comment:
libbase.a name didn't matter for a static lib, but i'm not sure about installing libbase.so to /usr/lib :)